change/@itwin-measure-tools-react-cf33bee1-ecf6-4c62-9114-68a6bad1fcdc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
is there a "none" type? if so lets do that, and remove the comment too, as these change files are user facing, and the api of the pkg didnt change, just the internals

Reply:
you actually want to change type not dependentChangeType

Reply:
should be good this time around.